packages/central-server/src/apiV2/meditrakApp/getChanges.js

```diff
@@ -33,10 +32,11 @@ function getRecordForSync(models, record, recordType, appVersion) {
     }
   });
 
+  const model = models.getModelForDatabaseRecord(recordType);
```

Comment:
This change is because we can get the correct model more accurately this way (e.g. for `user_account` database record type that i actually the `User` model)

Reply:
In this case, because in the meditrak sync queue I was calling these records `user_account`, what it was doing was looking for a model called `UserAccount` which doesn't exist